{"cells":[{"cell_type":"code","metadata":{"tags":[],"cell_id":"00000-f93b8b1f-e4bb-466f-90c7-157e541575cf","output_cleared":true},"source":["!dvc run --force -n gen_env -w .. -d data/default_env.pickle -p true_norm,colour_specific,shape_specific,target_area -o data/env.pickle python scripts/gen_env.py"],"execution_count":2,"outputs":[{"output_type":"stream","name":"stderr","text":"ERROR: dependency 'c:\\Users\\crast78p\\OneDrive - University of Otago\\PythonProjects\\my fork - learning_norms_with_mcmc_from_pcfg\\data\\default_env.pickle' does not exist\n"}]},{"cell_type":"code","source":["!dvc run --force -n gen_observations -w .. -d data/env.pickle -p target_area,true_norm,num_observations,obs_data_set -o data/observations.pickle python scripts/gen_observations.py"],"metadata":{"tags":[],"cell_id":"00001-16c1e51c-254a-43f2-86a9-f1eb33ff7515","output_cleared":false},"outputs":[{"name":"stdout","text":"Running stage 'gen_observations' with command:                        core\u001b[39m>\n\tpython scripts/gen_observations.py\nAdding stage 'gen_observations' in 'dvc.yaml'                         core\u001b[39m>\nUpdating lock file 'dvc.lock'\n\nTo track the changes with git, run:\n\n\tgit add dvc.yaml dvc.lock\n\u001b[0m","output_type":"stream"}],"execution_count":27},{"cell_type":"code","source":["!dvc run --force -n gen_mcmc_chains -w .. -d data/env.pickle -d data/observations.pickle -p n,m,rf,colour_specific,shape_specific,target_area -o data/chains.pickle --plots chain_likelihoods.csv python scripts/gen_mcmc_chains.py"],"metadata":{"tags":[],"cell_id":"00002-a888c5f1-50c2-4f53-911d-e56e17413bf5","output_cleared":false,"allow_embed":false},"outputs":[{"output_type":"stream","name":"stderr","text":"ERROR: '..\\chain_likelihoods.csv' is busy, it is being blocked by:\n  (PID 31140): C:\\Users\\crast78p\\OneDrive - University of Otago\\PythonProjects\\my fork - learning_norms_with_mcmc_from_pcfg\\env\\Scripts\\dvc run --force -n gen_mcmc_chains -w .. -d data/env.pickle -d data/observations.pickle -p n,m,rf,colour_specific,shape_specific,target_area -o data/chains.pickle --plots chain_likelihoods.csv python scripts/gen_mcmc_chains.py\n\nIf there are no processes with such PIDs, you can manually remove'.dvc/tmp/rwlock' and try again.\n"}],"execution_count":12},{"cell_type":"code","source":["!dvc repro"],"metadata":{"tags":[],"cell_id":"00002-62342832-eb77-412d-912e-5231b9148645"},"outputs":[{"output_type":"stream","name":"stderr","text":"ERROR: unexpected error - Stage: 'gen_env'\n\nHaving any troubles? Hit us up at https://dvc.org/support, we are always happy to help!\nDVC version: 1.7.9 (pip)\n---------------------------------\nPlatform: Python 3.8.2 on Windows-10-10.0.18362-SP0\nSupports: gdrive, http, https\nCache types: <https://error.dvc.org/no-dvc-cache>\nRepo: dvc, git\n"}],"execution_count":1},{"cell_type":"code","execution_count":null,"metadata":{},"outputs":[],"source":[]}],"nbformat":4,"nbformat_minor":2,"metadata":{"orig_nbformat":2,"deepnote_notebook_id":"406a8cf2-c2ac-4fed-9966-dbc145e270e1","deepnote_execution_queue":[{"cellId":"00002-a888c5f1-50c2-4f53-911d-e56e17413bf5","sessionId":"1feeeb0d-0e8e-4237-9cff-f9e097e69c59","msgId":"ca8934b6-4e53-4fb9-a2dc-f69ac4077aff"}],"kernelspec":{"name":"Python 3.8.2 32-bit ('env')","display_name":"Python 3.8.2 32-bit ('env')","metadata":{"interpreter":{"hash":"3caac4adb4e1f493f0617992483e3f4fe96c197b68af4c11de32cc1dac6ab887"}}}}}